recipe/meta.yaml

```diff
@@ -22,14 +24,53 @@ requirements:
   host:
     - ncurses
 
-test:
-  commands:
-    - test -f $PREFIX/lib/libestbase.so  # [linux]
-    - test -f $PREFIX/lib/libestbase.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libestools.so  # [linux]
-    - test -f $PREFIX/lib/libestools.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libeststring.so  # [linux]
-    - test -f $PREFIX/lib/libeststring.so.2.5  # [linux]
+outputs:
+  - name: libestools
+    script: install.sh  # [linux]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libestbase.so.{{ version }}  # [linux]
+        - test -L $PREFIX/lib/libestbase.so.2.5  # [linux]
+        - test -e $PREFIX/lib/libestbase.so.2.5  # [linux]
+        - test -f $PREFIX/lib/libestools.so.{{ version }}  # [linux]
+        - test -L $PREFIX/lib/libestools.so.2.5  # [linux]
+        - test -e $PREFIX/lib/libestools.so.2.5  # [linux]
+        - test -f $PREFIX/lib/libeststring.so.{{ version }}  # [linux]
+        - test -L $PREFIX/lib/libeststring.so.2.5  # [linux]
+        - test -e $PREFIX/lib/libeststring.so.2.5  # [linux]
+  - name: libestools-dev
+    script: install_devel.sh  # [linux]
+    requirements:
+      host:
+        - {{ pin_subpackage("libestools", exact=True) }}
```

Comment:
```suggestion
```
Since the dev package doesn't contain libraries, I don't think this host section is needed.